bin/asdf

```diff
@@ -27,7 +27,7 @@ find_asdf_cmd() {
   asdf_cmd_dir="$(asdf_dir)/lib/commands"
   case "$1" in
     'exec' | 'current' | 'env' | 'global' | 'install' | 'latest' | 'local' | \
-      'reshim' | 'uninstall' | 'update' | 'where' | 'which' | \
+      'reshim' | shell | 'uninstall' | 'update' | 'where' | 'which' | \
```

Comment:
Was there a problem when `shell` was not listed here?

Reply:
You want me to remove it?

Reply:
`shell` command has an escape hatch in an earlier call in the stack as linked as it may require exporting variables which is supported by calling https://github.com/asdf-vm/asdf/blob/master/lib/commands/command-export-shell-version.bash

I think this should be removed to avoid confusion.